### DISCUSSION TITLE
Ability to sort by player rank on the song selection screen

### DISCUSSION THREAD

**Discussion by copecone** [ppy/osu#24105] [Ideas] [answered]
As for the criteria for sorting the bitmap, I wish there was a function to sort the players currently logged into the client in the order of the rank achieved in that bitmap.
(English may be awkward because Google Translate is used)

**Comment by soopax** [comment]
Hello, 
sorry, I can't understand. Can you describe it with more detail please? Are you looking to sort beatmaps or players or scores?

**Reply by copecone** [reply]
Let me give you an example.
Let's assume we have an imaginary player A.

A plays map B. A achieved 1035511 points on map B and placed 24th.
This time A plays map C. A achieved 957711 points on map C and placed 49th.
That way he continued to play, finishing 35th on map D and 11th on map E.

Now, when A clicks the "ranking order" sort criteria in the osu!lazer client, E -> B -> D -> C are displayed in the order.

(I used Google Translate, so some expressions may be awkward)

**Comment by cihe13375** [comment]
Did you mean this?
![aaaaaaaa](https://github.com/ppy/osu/assets/33959010/b8993a35-c9c5-47d1-a1d6-a8bff826dedd)

**Reply by copecone** [reply]
no. I didn't mean to say that. Let me give you an example.
Let's assume we have an imaginary player A.

A plays map B. A achieved 1035511 points on map B and placed 24th.
This time A plays map C. A achieved 957711 points on map C and placed 49th.
That way he continued to play, finishing 35th on map D and 11th on map E.

Now, when A clicks the "ranking order" sort criteria in the osu!lazer client, E -> B -> D -> C are displayed in the order.

(I used Google Translate, so some expressions may be awkward)

**Comment by ghost** [comment, marked as answer]
Do you mean like this? Like rank achieved but for leaderboard spot?

![lbp](https://github.com/ppy/osu/assets/121861049/7f214235-f314-4561-8019-d6a14d2eb88c)

**Reply by copecone** [reply]
Yes, it's slightly different from what I was trying to say, but I was hoping for something like this!
